### DISCUSSION TITLE
Is there a way to trigger webhooks when deploy requests are created?

### DISCUSSION THREAD

**Discussion by DAlperin** [planetscale/discussion#182] [Q&A] [answered]
I am trying to make it so that a PR gets automatically opened when a deploy request does, but I can't seem to find a way to reliably be informed that the request was created. Is there an existing mechanism for this?

**Comment by fooforge** [comment, marked as answer]
Yes and no. We have plans for Webhooks, but I cannot say when we're going to get to it. As a stop gap, check out https://github.com/jonico/pscale-workflow-helper-scripts.

**Reply by DAlperin** [reply]
Yeah I saw that, it looks cool. The problem I have with that is I don't trust myself to consistently remember to ps-attach or create a PR. But happy to hear webhooks are coming eventually!
